xbmc/utils/test/TestStringUtils.h

```diff
@@ -0,0 +1,595 @@
+/*
```

Comment:
This is a good comment but could you add the standard copyright header before it?

Reply:
Nice catch. Will do. Also, why is copyright to 2018 and not 2022?